libs/service-portal/documents/src/screens/Overview/Overview.tsx

```diff
@@ -51,14 +51,16 @@ import {
   useDocumentTypesQuery,
 } from './DocumentExtra.generated'
 import { DocumentsPaths } from '../../lib/paths'
+import { useGetDocumentInboxLineQueryDetailsLazyQuery } from './DocumentDetail.generated'
 
 const pageSize = 10
 
 export const ServicePortalDocuments = () => {
   useNamespaces('sp.documents')
   const userInfo = useUserInfo()
   const { formatMessage } = useLocale()
-  const [page, setPage] = useState(1)
+  const loaderNumber = useLoaderData() as number
```

Comment:
scary

Reply:
Yeah, that's the go to solution for the useLoaderData with typescript. In this case not so scary as we only return a number always :D